front/app/hooks/useExceedsSeats.ts

```diff
@@ -0,0 +1,52 @@
+import useSeats from 'api/seats/useSeats';
+import { isNil } from 'utils/helperUtils';
+import { useAvailableSeats } from './useAvailableSeats';
+
+interface ReturnFunctionParameters {
+  newlyAddedAdminsNumber: number;
+  newlyAddedModeratorsNumber: number;
+}
+
+interface ExceedsSeats {
+  admin: boolean;
+  moderator: boolean;
+  any: boolean;
+  all: boolean;
```

Comment:
Minor: I wonder if maybe we could use the entire names for this to make them clearer.

```
  exceedsAdminSeats: boolean;
  exceedsModeratorSeats: boolean;
  exceedsModeratorOrAdminSeats: boolean;
  exceedsAdminAndModeratorSeats: boolean;
  ```
  
  They are longer but maybe easier to understand in this case

Reply:
It makes sense, but I think currently it's pretty clear when it's used like that `exceedsSeats.admin`. And I personally like grouping these booleans in one `exceedsSeats` object.

For me, the alternative looks a bit too long and verbose: `exceedsSeats.exceedsAdminSeats` or
```ts
  const {
    exceedsAdminSeats,
    exceedsModeratorSeats,
    exceedsModeratorOrAdminSeats,
    exceedsAdminAndModeratorSeats,
  } = useExceedsSeats();
```